src/javascripts/crosswords/crossword.js

```diff
@@ -791,7 +791,7 @@ class Crossword extends Component {
             <header className="instructions__title">
               Special instructions
             </header>
-            {this.instructions}
+            <span dangerouslySetInnerHTML={{ __html: this.instructions }} />
```

Comment:
Why the need for this change?

Reply:
This is to support the formatted instructions which we are planning to add to quick cryptic crosswords soon - I'm not a fan of blindly `dangerouslySetInnerHTML`ing the formatting - but that's what happens for clues https://github.com/guardian/react-crossword/blob/master/src/javascripts/crosswords/clues.js#L35 and we do have strict validation of the clues _and_ instructions text in the uploader.